Equilobe.DailyReport.Models/Storage/UserEngagementStats.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class UserEngagementStats
+    {
+        public long Id { get; set; }
+        public long InstalledInstanceId { get; set; }
```

Comment:
Do we need the instance Id? this is related to an AtlassianUser which is part of a single instance already.